product_docs/docs/biganimal/release/using_cluster/05c_upgrading_log_rep.mdx

```diff
@@ -0,0 +1,225 @@
+---
+title: Performing a major version upgrade of Postgres on BigAnimal
```

Comment:
Additional:

Global recommended edits:
Use “source / target” rather than “old / new” when referring to instances.
Use an ordered list for the parent steps (Gather Info, Confirm Version, Migrate Schema, etc…)
Each parent step should use sub-steps (a,b,c…)
Each parent topic should use “ing”.  (Confirming, Migrating, etc…)
Use “BigAnimal instance” in place of “instance”
Avoid using ‘Please’

Reply:
Is it in the styleguide that we should always use gerunds in topics?

Reply:
From style guide:  Procedure headings use gerunds, for example, Modifying your cluster.

Reply:
thanks, will adjust accordingly